src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -632,6 +632,11 @@ want to solicit feedback from the community.
 
     Added in `1.8.0`.
 
+    When set to `disk` along with setting
```

Comment:
As a user, I think I would have a lot of questions about this feature. What are the file names? What is the file structure? What is the retention policy? Are there multiple files? What do these files mean? How are they managed? How are they cleaned up? Stuff like that. I think we may need more docs. Doesn't have to be on this PR.

Reply:
+1, in addition, who has access to these files, any security/privacy concern, etc.

Reply:
Agree we need to explain these.

I think it would be better to improve the doc here to cover these details: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.PersistentStorage.FileSystem.  We could then link the details.

Reply:
Code for persistent storage is vendored in this exporter, and it does not use a persistent storage package reference. It’s appropriate to document this here.

Reply:
We have a process defined: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/PersistentStorage/README.md#persistent-storage-apis-for-otlp-exporter. Currently we are not forking Readmes, but I think we should do that since we do not make any customizations to the forked files.